config.json

```diff
@@ -127,9 +127,7 @@
         "binaries":{
           "description": "Only builds binaries. It doesn't restore packages.",
           "settings":{
-            "RestoreDuringBuild": false,
-            "BuildPackages": false,
-            "BuildTests": false
+            "RestoreDuringBuild": false
```

Comment:
Seems like this change is not particular to the rest of the work you are doing. I wonder if it's better to have it in it's own PR to better explain why do we need this change.